AutomatedTesting/Gem/PythonTests/scripting/scripting_utils/scripting_tools.py

```diff
@@ -411,6 +411,7 @@ def located_expected_tracer_lines(self, section_tracer, lines):
     for line in lines:
         for found_line in found_lines:
             if line == found_line:
+                print("found line: " + found_line)
```

Comment:
If this was just used for debugging it can be removed.

Reply:
this is removed in the PR following this one https://github.com/o3de/o3de/pull/10870